### PR TITLE
Update Java wizard pages to SDK 3.0.

### PR DESCRIPTION
**Pull request by maciejwalkowiak**

### CODE REVIEW

**Comment by maciejwalkowiak** [comment]
cc @bruno-garcia

File: src/wizard/java/index.md [hunk 1]
```diff
@@ -13,131 +13,127 @@ Using Maven:
...
 
 ```scala
-libraryDependencies += "io.sentry" % "sentry" % "{{ packages.version('sentry.java', '1.7.30') }}"
+libraryDependencies += "io.sentry" % "sentry" % "{{ packages.version('sentry.java', '3.0.0') }}"
 ```
 
 ## Capture an Error
 
-To report an event manually you need to initialize a `SentryClient`. It is recommended that you use the static API via the `Sentry` class, but you can also construct and manage your own `SentryClient` instance. An example of each style is shown below:
+To report an event manually you need to initialize a `Sentry` class. It is recommended that you use the static API via the `Sentry` class, but you can also construct and manage your own `IHub` instance. An example of each style is shown below:
```
Comment (bruno-garcia) [inline src/wizard/java/index.md:34]: Not sure this change makes sense.
The doc here is trying to say you can do it either with SentryClient or with the static Sentry class. We can't "initialize a Sentry class", it talks about Sentry init in the second phrase.

Maybe we delete the whole sentence?

**Review by bruno-garcia — COMMENTED** [review]
Lets do this on a call

**Comment by maciejwalkowiak** [comment]
Lets start over again.

**Pull request closed without merging**